### DISCUSSION TITLE
The JWT Token don't contain the securityId property.

### DISCUSSION THREAD

**Discussion by alvis-do** [loopbackio/loopback-next#7427] [Q&A] [answered]
I am following a JWT authentication document, but when I check the value of the generated JWT token. I don't see the security property, which means this property is undefined. 
```
const user = await this.userService.verifyCredentials(loginReq.provider);

const userProfile = this.userService.convertToUserProfile(user);
console.debug(userProfile[securityId]) // Result: A string value. OK.

const token = await this.jwtService.generateToken(userProfile);
// console.debug inner the generateToken function and Result: An undefined value.
```

### Expect:
The JWT Token generated from `jwtService.generateToken` function contains an Id property.

Does anybody help me fix this? please

**Comment by dougal83** [comment, marked as answer]
Hi @kudos-cmd 

If you take a look at the source you will see the `securityId` is given to the `id` property in the `generateToken` function:

https://github.com/strongloop/loopback-next/blob/70f37d7cfc09db01ed5cf49c3150ce32ec16fcf2/extensions/authentication-jwt/src/services/jwt.service.ts#L54-L77

Try accessing the `id` property in this case to access the user's Id.  You might also expect to see the property named `sub` to contain the user (subject) id but that is not the case here.

**Reply by dougal83** [reply]
Forgot to mention if that is no help then post an example of your code in a public repository and post a link for someone to take a look.
